whist/core/deck.py

```diff
@@ -29,7 +29,7 @@ def add(self, card: Card) -> NoReturn:
         :param card: card to add
         """
         if card in self.__cards:
-            raise KeyError
+            raise KeyError(f'{card.name} already in deck')
```

Comment:
```suggestion
            raise KeyError(f'{card} already in deck')
```
__str__ is implemented

Reply:
I wasn't sure if it works in `f'{}'`